configuration-management.yaml

```diff
@@ -0,0 +1,302 @@
+openapi: "3.0.0"
+info:
+  description: "This is the Configuration API for UC4."
+  version: "0.10.1"
+  title: "UC4"
+
+servers:
+  - url: https://uc4.cs.upb.de/api/production/configuration-management
+
+tags:
+- name: "Version Management"
+  description: "Everything about the Version"
+  externalDocs:
+    description: "Find out more"
+    url: "https://github.com/upb-uc4"
+- name: "Configuration Management"
+  description: "Everything about the configuration"
+  externalDocs:
+    description: "Find out more"
+    url: "https://github.com/upb-uc4"
+
+paths:
+  /version:
+    get:
+      tags:
+      - "Version Management"
+      summary: "Get version"
+      description: "Returns the version of the current service"
+      operationId: "getVersion"
+      responses:
+        "200":
+          description: "OK"
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/JsonVersionNumber'
+  /version/hyperledger:
+    get:
+      tags:
+      - "Version Management"
+      summary: "Get versions of the hyperledger api, chaincode and network"
+      description: "Get versions of the hyperledger api, chaincode and network of the underlying hyperledger fabric network"
+      operationId: "getVersionHyperledger"
+      responses:
+        "200":
+          description: "OK"
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/HyperledgerVersions'
+              examples:
+                HyperledgerVersionsExample:
+                  $ref: '#/components/examples/HyperledgerVersions'
```

Comment:
just for my information: what happens when you can not fetch one of the versions? how does the json look then, is one version set to undefined? null? unavailable? Could you add an example to this documentation?

Reply:
Failure in the backend to procure the hyperledger versions will result in an internal server error